js/polyfills/crypto.js

```diff
@@ -0,0 +1,235 @@
+// Polyfill for crypto and tinygo wasm
+if (!globalThis.crypto) {
+    globalThis.crypto = global.crypto = {
```

Comment:
@brunocalza This file has the 2 extra spaces as well.  Looks like it's not being linted, which is good since it's not written by use exactly.  Should we remove the extra two spaces?